corehq/apps/registration/static/registration/js/server_location_select.js

```diff
@@ -0,0 +1,36 @@
+import ko from "knockout";
```

Comment:
It's possible this javascript component could be simplified a bit now that I'm not sharing it between both signup and login forms. However, there are planned near-future changes to the cloud location selection widget on the registration form anyway, which may either require more extensive changes to this javascript or potentially getting rid of this file all together.